src/main/java/bio/terra/workspace/service/workspace/exceptions/CloudContextRequiredException.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.workspace.service.workspace.exceptions;
+
+import bio.terra.workspace.common.exception.BadRequestException;
+
+/** When you can't get there from here, but somehow end up there */
+public class CloudContextRequiredException extends BadRequestException {
+  public CloudContextRequiredException(String message) {
```

Comment:
fwiw, IJ will give you all 4-5 overloads of the constructor so you have flexibility in declaring these.